v03_pipeline/lib/misc/family_loading_failures_test.py

```diff
@@ -202,7 +202,6 @@ def test_get_families_failed_sex_check(self):
             [
                 [
                     'Sample ROS_006_18Y03226_D1 has pedigree sex F but imputed sex M',
-                    'Sample ROS_006_18Y03227_D1 has pedigree sex M but imputed sex F',
```

Comment:
the logic behind this change... `ROS_006_18Y03227_D1` is now denoted as `U` in the test pedigree, so the family fails for only the `'Sample ROS_006_18Y03226_D1 has pedigree sex F but imputed sex M'` reason.